solr/FULLSTORY-CHANGES.txt

```diff
@@ -3,4 +3,5 @@
 * fs.ignoreconflicts (Scott Blum)
 * FullStory: special OOMKILLER handling for spaces (Scott Blum)
 * Remove pathological error reporting. Don't serialize the entire clusterstate to log (Scott Blum)
-* Expose DocValuesTermsCollector and TermsQuery (Jaime Yap)
\ No newline at end of file
+* Expose DocValuesTermsCollector and TermsQuery (Jaime Yap)
```

Comment:
just DocValuesTermsCollector now?

Reply:
Well , we don't need `DocValuesTermsCollector` anymore.